web/src/zform.js

```diff
@@ -1,43 +1,68 @@
 import $ from "jquery";
+import {z} from "zod";
```

Comment:
This looks like another instance of import-js/eslint-plugin-import#2293; you’ll need to add it to the ignore list for the `import/no-unresolved` rule in `.eslintrc.json`.